lab-andrew/server.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const http = require('http');
+const parseUrl = require('url').parse;
+const parseQuery = require('querystring').parse;
+const cowsay = require('cowsay');
+const parseBody = require('./lib/parse-text');
+
+const port = process.argv[2] || 3000;
+
+const server = http.createServer(function(req, res) {
+  req.url = parseUrl(req.url);
+  req.url.query = parseQuery(req.url.query);
+
+
+
+  if (req.url.pathname === '/') {
+    res.writeHead(200, {'Content-Type': 'text/plain'});
+    res.write('API endpoints: \n/cowsay');
+    res.end();
+    return this;
+  }
+
+
+  if(req.url.pathname === '/cowsay' && req.method === 'POST') {
```

Comment:
This route declaration is good, you'll want to add a '/api/cowsay' route which will function exactly the same way as '/cowsay'.